asf_heat_pump_suitability/getters/get_target.py

```diff
@@ -0,0 +1,183 @@
+import polars as pl
+
+from asf_heat_pump_suitability import config
+from asf_heat_pump_suitability.getters import base_getters
+
+
+def get_df_target_nrooms() -> pl.DataFrame:
+    """
+    Get dataframe of counts of total number of rooms for properties in all LSOAs in England and Wales. Source: census data 2021.
+
+    Returns:
+        pl.Dataframe: counts of total number of rooms for properties in all LSOAs in England and Wales
+    """
+    df = pl.read_csv(config["data_source"]["EW_census_number_of_rooms"])
+    df = df.drop(
+        [
+            "Lower layer Super Output Areas",
+            "Number of rooms (Valuation Office Agency) (9 categories)",
+        ]
+    )
+    df = df.rename(
+        {
+            "Lower layer Super Output Areas Code": "lsoa",
+            "Number of rooms (Valuation Office Agency) (9 categories) Code": "number_of_rooms",
+        }
+    )
+    return df.pivot(
+        index="lsoa", columns="number_of_rooms", values="Observation"
+    ).rename({"9": "9+"})
+
+
+def get_df_target_property_type(censored_values: int = 0) -> pl.DataFrame:
+    """
+    Get dataframe of property type counts for all LSOAs in England and Wales, and fill censored values with given
+    constant. Source: census data 2021.
+
+    Args:
+        censored_values (int): value to fill censored values with, 0-9. Default 0.
+
+    Returns:
+        pl.Dataframe: counts of property type for all LSOAs in England and Wales
+    """
+    content = base_getters.get_content_from_path(
+        config["data_source"]["EW_census_housing_characteristics"]
+    )
+    df = pl.read_excel(content, sheet_name="2c", engine="calamine")
+
+    # Remove empty header rows
+    df = df.rename(df[2].to_dicts().pop()).slice(
+        3,
+    )
+    df = df.rename({"Area Code": "lsoa"}).drop(["Area Name"])
+
+    return _fill_df_censored_values(df, censored_values)
```

Comment:
Again, I would suggested putting this operation before, mostly for de-bugging reasons!